client/me/purchases/purchase-item/index.jsx

```diff
@@ -399,7 +444,7 @@ class PurchaseItem extends Component {
 									event.preventDefault();
 									page( getPurchaseListUrlFor( slug ) );
 								} }
-								title={ translate( 'View subscriptions for %(siteName)s', {
+								title={ translate( 'View subscriptions for %(siteDomain)s', {
 									args: {
 										siteName: site.name,
```

Comment:
Don't forget to change this to `siteDomain` also.

Reply:
Fixed. Thanks for the catch!